.circleci/config.yml

```diff
@@ -1459,6 +1459,27 @@ workflows:
               - install-tools
               - attach_workspace:
                   at: /
+  continuous-integration-gcp-restart-misc:
```

Comment:
Will this run on the some label ?

Reply:
How to associate it with a github label ?

Reply:
As discussed we can run these tests on PR commits.